goss/goss.yaml

```
@@ -38,6 +38,11 @@ command:
     exit-status: 0
     stdout:
       - 18.18.2
+  playwright_version:
+    exec: "playwright --version'" # Playwright is installed as asdf shims for `jenkins` user only
```

```suggestion
    exec: playwright --version
```

There was a minor typo causing the goss failure (trailing simple quote `'`).

I'm merging the suggestion above (with removal of double quotes and trailing comment to make the comment easier to get)